.changeset/slimy-turtles-look.md

```
@@ -0,0 +1,93 @@
+new PineconeVector({ id: 'my-vector', apiKey: '...', controllerHostUrl: '...' })
+```
+
+**@mastra/clickhouse**
+
+Added support for all `ClickHouseClientConfigOptions` like `request_timeout`, `compression`, `keep_alive`, and `database`. Existing configurations continue to work unchanged.
+
+**@mastra/cloudflare, @mastra/cloudflare-d1, @mastra/lance, @mastra/libsql, @mastra/mongodb, @mastra/pg, @mastra/upstash**
+
+Improved logging by replacing `console.warn` with structured logger in workflow storage domains.
+
+**@mastra/deployer-cloud**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)